BWA_CHIP/submit_BWA_CHIP.sh

```diff
@@ -26,11 +49,32 @@ else
     array_length=$(wc -l < ${fastq_list}) #get the number of FASTQs 
 
     sbatch -o "${output_directory}/Logs/%A_%a.log" `#put into log` \
-        -a "1-${array_length}" `#initiate job array equal to the number of fastq files` \
-        "${code_directory}/BWA_CHIP.sh" \
+            -a "1-${array_length}" `#initiate job array equal to the number of fastq files` \
```

Comment:
Can we make this indent just one tab in from the line above?  It got shifted and the alignment looks weird.